src/metabase/query_processor/util.clj

```diff
@@ -105,7 +95,7 @@
     (cond-> query
       (empty? constraints) (dissoc :constraints)
       (empty? parameters)  (dissoc :parameters)
-      true                 remove-lib-uuids
+      true                 lib.schema.util/remove-lib-uuids
```

Comment:
I wanted to move this to `lib.util` instead of `lib.schema.util`, but that created a circular import dependency since `lib.util` indirectly requires stuff from `lib.schema.util`, and `lib.schema.util` wants to call this function.

Reply:
I also considered putting it in `metabase.util`, but didn't seem like the right place.